lib/tasks/setup.rake

```diff
@@ -1,11 +1,25 @@
 namespace :setup do
+
   task :run, [:environment] do
```

Comment:
Don't know. I didn't mess with the setup file, when I pulled from the origin I had to resolve a merge conflict, that's what I ended up with. Should I change it back to :migrate?